src/app/example_local.ml

```diff
@@ -14,7 +14,10 @@ let configuration =
     section "Additional Services" [
       env "tls_port" ~required:false ~example:"22443"
         ~help:"Force the use of a TLS tunnel exposed at a given port (highly \
-               recommended when using a publicly visible instance)."
+               recommended when using a publicly visible instance).";
+      env "epidisco_dev" ~required:false ~example:"/path/to/shared/directory"
```

Comment:
This is great -- every time I set up a new instance, I realize we could do better to get people going with epidisco, but I always kept forgetting to mention/fix this :\